README.md

```diff
@@ -87,9 +96,10 @@ const PublicReportingAPI = require('agent-js-jasmine/lib/publicReportingAPI');
 describe('A suite', function() {
     PublicReportingAPI.addAttributes([{
         key: 'suiteKey',
-        value: 'suiteValue'
+        value: 'suiteValue',
+        suite: 'A suite',
     }]);
-    PublicReportingAPI.addDescription('Suite description');
+    PublicReportingAPI.addDescription({ text: 'Suite description', suite: 'A suite' });
```

Comment:
I suggest to keep the single signature of `addDescription` method. Just add optional suite parameter for reporting description to the suite. 
`PublicReportingAPI.addDescription('Suite description', 'A suite');` - for the suite
`PublicReportingAPI.addDescription('Test description');` - for the test
What do you think?